storybook/qmlTests/tests/tst_DAppsWorkflow.qml

```diff
@@ -131,4 +350,168 @@ Item {
             verify(eip155.events.length > 0)
         }
     }
+
+    // Beware this TestCase should be last; I had it before ServiceHelpers and it was not run with `when: windowShown`
```

Comment:
@micieslak regarding my issue of not running this TestCase that has `when: windowShown`. After more trial and error I found out that if I move it last, after `ServiceHelpers`, it will run anyway. Thanks again for you help.